src/gitmojify/mojify.py

```diff
@@ -60,7 +60,7 @@ def gitmojify(
         The gitmojified message.
     """
     if any(map(message.startswith, convert_prefixes or [])):
-        first_word, *rest = message.split()
+        first_word, *rest = message.split(maxsplit=1)
         if first_word.endswith(":"):
             first_word = first_word[:-1]
         message = f"{first_word.lower()}: {' '.join(rest)}"
```

Comment:
We don't need to join the rest on space in this case anymore.

```suggestion
        message = f"{first_word.lower()}: {rest}"
```

Reply:
When using `*rest` to capture other output, the result is a list and it may be empty.
Since the count will only be 0 or 1, instead of using join, it might be written like this?

```py
f"{first_word.lower()}: {rest[0] if rest else ''}"
```

Reply:
Indeed, you're right. Let's just keep the join in that case, so we don't need a condition. But we can just join on `''` (no space).